lib/api/envs.json

```diff
@@ -3,8 +3,8 @@
     "name": "Localhost",
     "variables": [
       {
-        "key": "host",
-        "value": "http://localhost:4000"
+        "value": "http://localhost:4000",
```

Comment:
Is this file auto-generated? seems like this is change for the sake of change. 🤷‍♂️

Reply:
I'm exporting the environments everytime there's a change. So it's not "a change for the sake of change"

Reply:
Sorry, not implying that you were the one making a change just for the sake of it. Just that the diff in this PR _appears_ that way. The variable key & value are the same.